Scroller/EditableWrapper.js

```diff
@@ -283,21 +291,39 @@ const EditableWrapper = (props) => {
 	const handleMouseMove = useCallback((ev) => {
 		const {centeredOffset, itemWidth, selectedItem} = mutableRef.current;
 		if (selectedItem) {
+			mutableRef.current.mouseClientX = ev.clientX;
 			// Determine toIndex with mouse client x position
 			const toIndex = Math.floor((ev.clientX + scrollContentRef.current.scrollLeft - centeredOffset) / itemWidth);
 
+			mutableRef.current.lastInputType = 'mouse';
 			moveItems(toIndex);
 		}
 	}, [moveItems, scrollContentRef]);
 
 	const handleMouseLeave = useCallback(() => {
-		const {selectedItem} = mutableRef.current;
+		const {itemWidth, lastInputType, mouseClientX, selectedItem} = mutableRef.current;
+		const scrollContentNode = scrollContentRef.current;
+		const scrollContentCenter = scrollContentNode.getBoundingClientRect().width / 2;
+
 		if (selectedItem) {
 			const orders = finalizeOrders();
 			forwardCustom('onComplete', () => ({orders}))({}, editable);
 			reset();
+
+			if (lastInputType === 'scroll') {
+				let left = 0;
```

Comment:
How about setting this to `scrollContentNode.scrollLeft` and..

Reply:
Let's use the ternary operator instead.

Reply:
I fixed to use ternary operator instead.